Tools/clinic/clinic.py

```diff
@@ -2474,9 +2492,8 @@ def __init__(
         self.modules: ModuleDict = {}
         self.classes: ClassDict = {}
         self.functions: list[Function] = []
-        # dict: include name => reason
-        # Example: 'pycore_long.h' => '_PyLong_UnsignedShort_Converter()'
-        self.includes: dict[str, tuple[str, str]] = {}
+        # dict: include name => Include instance
```

Comment:
Does this need to be a dictionary anymore, or could it just be a `list[Include]`?

Reply:
Yes. It's a dict so Clinic.add_include() has a complexity of O(1).